### PR TITLE
Adyen: add Maestro and Electron support

### PR DESCRIPTION
**Pull request by bpollack**
Unit: 23 tests, 109 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

Remote: 35 tests, 88 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

### CODE REVIEW

File: test/remote/gateways/remote_adyen_test.rb [hunk 1]
```diff
@@ -17,6 +17,16 @@ def setup
...
     @declined_card = credit_card('4000300011112220')
 
+    @maestro = credit_card(
+      '5500000000000004',
+      month: 8,
+      year: 2018,
+      first_name: 'John',
+      last_name: 'Smith',
+      verification_value: '737',
+      brand: 'mastercard'
```
Comment (curiousepic) [inline test/remote/gateways/remote_adyen_test.rb]: Isn't maestro is a legit brand in AM?

Comment (bpollack) [reply test/remote/gateways/remote_adyen_test.rb]: It is, but this feature is explicitly testing the ability to _override_ the brand via those fields for use in Core; this is intentional. Ping me on Slack and I can link you to the Zendesk ticket for more background.

Comment (bpollack) [reply test/remote/gateways/remote_adyen_test.rb]: Although this variable should be changed to `@mastercard`; let me do that.

Comment (curiousepic) [reply test/remote/gateways/remote_adyen_test.rb]: I see. Perhaps the commit/changelog wording could be more accurate re:overriding?

**Comment by bpollack** [comment]
@curiousepic Tweaked naming of variables/changelog to be for the AM audience, not the Spreedly audience.

**Pull request merged**